File: Selenium/Activities/Activity19.java
package activites;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.Select;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.time.Duration;


public class Activity19 {
    public static void main(String[] args) {

        WebDriverManager.firefoxdriver().setup();

        WebDriver driver = new FirefoxDriver();

        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));


        driver.get("https://www.training-support.net/selenium/javascript-alerts");

        System.out.println("Home page title: " + driver.getTitle());


        driver.findElement(By.id("confirm")).click();


        Alert confirmAlert = driver.switchTo().alert();


        String alertText = confirmAlert.getText();
        System.out.println("Text in alert: " + alertText);


        confirmAlert.accept();

        driver.quit();
    }
}

File: Selenium/Activities/Activity19.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:
    driver.get("https://www.training-support.net/selenium/javascript-alerts")

    print("Page title is: ", driver.title)

    driver.find_element(By.ID, "confirm").click()

    confirm_alert = driver.switch_to.alert

    print(confirm_alert.text)

    confirm_alert.accept()
